assets/csidriveroperators/aws-ebs/hypershift/mgmt/configmap_role.patch.yaml

```diff
@@ -0,0 +1,18 @@
+- op: "add"
+  path: "/rules/-"
+  value:
+    apiGroups:
+      - ''
+    resourceNames:
+      - extension-apiserver-authentication
+      - aws-ebs-csi-driver-operator-lock
+    resources:
+      - configmaps
```

Comment:
generated/rbac.authorization.k8s.io_v1_role_aws-ebs-csi-driver-operator-role.yaml already has permissions to read/update _any_ ConfigMap in the operator namespace.

Reply:
removed.